PWGHF/D2H/DataModel/ReducedDataModel.h

```diff
@@ -173,6 +186,43 @@ DECLARE_SOA_COLUMN(MlScoreNonpromptMassHypo0, mlScoreNonpromptMassHypo0, float);
 DECLARE_SOA_COLUMN(MlScoreBkgMassHypo1, mlScoreBkgMassHypo1, float);             //! ML score for background class (mass hypothesis 1)
 DECLARE_SOA_COLUMN(MlScorePromptMassHypo1, mlScorePromptMassHypo1, float);       //! ML score for prompt class (mass hypothesis 1)
 DECLARE_SOA_COLUMN(MlScoreNonpromptMassHypo1, mlScoreNonpromptMassHypo1, float); //! ML score for non-prompt class (mass hypothesis 1)
+DECLARE_SOA_COLUMN(MinNumItsClsProng, minNumItsClsProng, int);                   //! minimum value of number of ITS clusters for the decay daughter tracks
+DECLARE_SOA_COLUMN(MinNumTpcCrossedRowsProng, minNumTpcCrossedRowsProng, int);   //! minimum value of number of TPC crossed rows for the decay daughter tracks
+DECLARE_SOA_COLUMN(MinAbsEtaProng, minAbsEtaProng, float);                       //! minimum value of absolute pseudorapidity for the decay daughter tracks
+DECLARE_SOA_COLUMN(MinPtProng, minPtProng, float);                               //! minimum value of transverse momentum for the decay daughter tracks
+DECLARE_SOA_COLUMN(MaxChi2Tpc, maxChi2Tpc, float);                               //! maximum value of TPC chi2 for the decay daughter tracks
```

Comment:
```suggestion
DECLARE_SOA_COLUMN(NItsClsProngMin, nItsClsProngMin, int);                   //! minimum value of number of ITS clusters for the decay daughter tracks
DECLARE_SOA_COLUMN(NTpcCrossedRowsProngMin, nTpcCrossedRowsProngMin, int);   //! minimum value of number of TPC crossed rows for the decay daughter tracks
DECLARE_SOA_COLUMN(EtaAbsProngMin, etaAbsProngMin, float);                       //! minimum value of absolute pseudorapidity for the decay daughter tracks
DECLARE_SOA_COLUMN(PtProngMin, ptProngMin, float);                               //! minimum value of transverse momentum for the decay daughter tracks
DECLARE_SOA_COLUMN(Chi2TpcMax, chi2TpcMax, float);                               //! maximum value of TPC chi2 for the decay daughter tracks
```

Reply:
Done (slightly differently, to match the names in the Analysis DataModel for track extensions)